go/interpreter/lfvm/instructions.go

```diff
@@ -760,8 +756,8 @@ func selfDestructNewAccountCost(accountExists bool, balance tosca.Value) tosca.G
 }
 
 func selfDestructRefund(destructed bool, revision tosca.Revision) tosca.Gas {
+	// Since London and after there is no more refund (see https://eips.ethereum.org/EIPS/eip-3529)
```

Comment:
Thanks for looking this up. To match the comment, the line below should be `< tosca.R10_London`.